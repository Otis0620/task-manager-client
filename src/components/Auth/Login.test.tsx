import React from 'react';
import { shallow } from 'enzyme';
import Login from './Login';

test('Login', () => {
  const wrapper = shallow(<Login />);

  expect(wrapper).toMatchSnapshot();
});
