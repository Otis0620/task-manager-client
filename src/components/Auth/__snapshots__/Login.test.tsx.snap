// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login 1`] = `
<div
  className="container"
>
  <div
    className="form-container"
  >
    <div
      className="form"
      onSubmit={[Function]}
    >
      <div>
        <h2>
          Login
        </h2>
        <input
          className="input"
          name="email"
          onChange={[Function]}
          placeholder="Email"
          type="email"
          value=""
        />
        <input
          className="input"
          name="password"
          onChange={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
        <input
          onClick={[Function]}
          type="submit"
          value="Submit"
        />
      </div>
    </div>
  </div>
</div>
`;
